extensions/vue/languages/vue/injections.scm

```diff
@@ -1,7 +1,54 @@
-(script_element
-  (raw_text) @content
+; <script lang="js">
+((script_element
+  (start_tag
+    (attribute
+      (attribute_name) @_lang
+      (quoted_attribute_value
+        (attribute_value) @_js)))
+  (raw_text) @content)
+  (#eq? @_lang "lang")
+  (#eq? @_js "js")
```

Comment:
If I understand correctly, I think that this change would break the JS injection for plain `<script>` tags, ie those without a `lang="..."` attribute.

Also, this may want to use `(#any-of? @_js "js" "javascript")` to handle different ways of tagging JS. Ditto below for `"ts" "typescript"` etc. Or maybe that's not valid and Vue only supports `js` and `ts`?